frontend/src/pages/NewTeacher.vue

```diff
@@ -0,0 +1,413 @@
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          PhoneNumber: this.PhoneNumber,
+          FullName: this.name,
+          Source: this.source,
+          NativeLanguageString: this.selected.NativeLanguage, 
+          EnglishProficiency: this.EnglishProficiency, 
+          Notes: this.Notes,
+          StatusString: "SCREENING"
+        })
+      }
+      fetch("/api/teachers", teacherCreate)
+        .then(response => {
+          if (response.status < 400) {
+            this.$buefy.notification.open({
+              message: 'New teacher added. <u>View profile</u>!',
+              duration: 3000,
+              type: 'is-success',
+              position: 'is-top',
+            })
+            // refreshes state
+            this.getNativeLanguages()
+            setTimeout(() => {this.$router.push({path: `/teachers`})}, 5000)} 
+          else {
+            this.$buefy.notification.open({ 
+              message: 'Something went wrong. Please try again.',
+              duration: 3000, 
+              type: 'is-warning',
+              position: 'is-top'
+            })
+          }
+       })
+    },
+    
+    filteredLanguageDataArray(language = "") {
+      this.languages = this.API_nativeLanguage.filter((option) => {
+          return option.NativeLanguage
+            .toLowerCase()
+            .includes((language || "").toLowerCase())
+          
+      })
+    },
+
+    // Auto complete for second langauge not working 
+
+    filteredSecondLanguageDataArray(language = "") {
+      this.languages = this.API_nativeLanguage.filter((option) => {
+          return option.SecondLanguage
+            .toLowerCase()
+            .includes((language || "").toLowerCase())
+          
+      })
+    },
+
+
+    uploadFile(){
+      this.$buefy.notification.open({
+        message: 'The file was uploaded successfully!',
+        duration: 5000,
+        type: 'is-success',
+        position: 'is-top',
+      })
+    },
+
+
+    errorUpload(){
+      this.$buefy.notification.open({
+        message: '<b>There was an error in uploading.</b> <br> Please check formatting of the file and try again.',
+        duration: 5000,
+        type: 'is-danger',
+        position: 'is-top',
+      })
+    },
+
+
+    showAddLanguage() {
+      this.$buefy.dialog.prompt({
+        message: `Add new language`,
+        inputAttrs: {
+          placeholder: 'e.g. Italian',
+          maxlength: 255,
+        },
+        confirmText: 'Add',
+        onConfirm: async (value) => {
+          this.selected.NativeLanguage = value
+        }
+      })
+    },
+    
+    
+
+    showAddSecondLanguage() {
+      this.$buefy.dialog.prompt({
+        message: `Add new language`,
+        inputAttrs: {
+          placeholder: 'e.g. Italian',
+          maxlength: 255,
+        },
+        confirmText: 'Add',
+        onConfirm: (value) => {
+          this.$refs.secondlanguagevalue.setSelected(value)
```

Comment:
Your onConfirm here and the onConfirm for the first language look quite different? If the first one is working I suggest following that, or perhaps referring to the documentation to see which is the correct way to set the value after it's been entered.